wagtail/wagtailadmin/views/generic.py

```diff
@@ -71,6 +71,10 @@ def get(self, request):
         if self.context_object_name:
             context[self.context_object_name] = object_list
 
+        return context
+
+    def get(self, request):
+        context = self.get_context()
```

Comment:
This change conflicts with (and breaks) the default list view handling introduced in #3625. See [default Django behavior here](https://github.com/django/django/blob/1.11.7/django/views/generic/list.py#L159) that supports both `self.template_name` and `self.get_template_names()` as possible template sources.